frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDashboard.tsx

```diff
@@ -13,9 +17,12 @@ const TeamEventCreditDashboard = (props: {
   approvedAttendance: TeamEventAttendance[];
   pendingAttendance: TeamEventAttendance[];
   isAttendanceLoading: boolean;
+
+  // teamEventAttendance: TeamEventAttendance;
 }): JSX.Element => {
   const { allTEC, approvedAttendance, pendingAttendance, isAttendanceLoading } = props;
```

Comment:
You also need to unpack `setPendingAttendance` from the props parameter, i.e.

Before:
```
const { allTEC, approvedAttendance, pendingAttendance, isAttendanceLoading } = props;
```

After:
```
const { allTEC, approvedAttendance, pendingAttendance, isAttendanceLoading, setPendingAttendance } = props;
```